css/main.css

```diff
@@ -0,0 +1,593 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&family=Inter:wght@400;700&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+a {
+  text-decoration: none;
+}
+
+hr {
+  width: 80vw;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 15px;
+  background-color: #3c3a39;
+  color: white;
+}
+
+#desktop-nav,
+#desktop-nav ul {
+  display: none;
+}
+
+#logo {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 24px;
+}
+
+.menu-bars {
+  background: transparent;
+  border: none;
+  outline: none;
+}
+
+.menu-bars i {
+  width: 18px;
+  height: 12px;
+  color: white;
+}
+
+#mobile-nav {
+  z-index: 9999;
+  position: absolute;
+  background-color: black;
```

Comment:
Please give the `#modile-nav` the right `background-color` because it is different than the Figma design:

- Your background is black
- Figma's background is:
![background](https://user-images.githubusercontent.com/95412943/210756143-262595d3-5cd5-4e28-a3ad-2343d7c40b65.PNG)

Reply:
You're right. I'll fix ir